docs/modules/plugins/pages/plugin-mobile-app.adoc

```diff
@@ -326,6 +326,9 @@ The atomic actions performed are:
 * type text into the element
 * hide keyboard
 
+[WARNING]
+For `iOS` platform the hide keyboard action works only for XCUIElementTypeTextField element and skipped for XCUIElementTypeTextView element, in case of XCUIElementTypeTextView element use tap step to tap outside the text view.
+
```

Comment:
```
For `iOS` platform the hide keyboard action works only for `XCUIElementTypeTextField` elements and skipped for `XCUIElementTypeTextView` elements, in case of `XCUIElementTypeTextView` element tap outside the text view to close the keyboard.
```

Reply:
Also should there be reference that this is applicable only for real devices?